index.html

```diff
@@ -3743,21 +3743,19 @@ <h3>Discovery Examples</h3>
       <p>
         The following example finds {{ThingDescription}} objects of <a>Thing</a>s that are exposed by local hardware, regardless how many instances of <a>WoT Runtime</a> it is running. Note that the discovery can end (become inactive) before the internal <a>discovery results</a> queue is emptied, so we need to continue reading {{ThingDescription}} objects until done. This is typical with local and directory type discoveries.
       </p>
-      <pre class="example" title="Discover Things exposed by local hardware">
-        let discovery = new ThingDiscovery({ method: "local" });
+      <pre class="example" title="Fetch the Thing Description of a Thing">
```

Comment:
Is the explainer text above of this example updated? It refers to local discovery with multiple WoT Runtime a concept that I think we are still missing to formalize.